dispatch/apps/api/exceptions.py

```diff
@@ -4,3 +4,7 @@
 class InvalidFilename(APIException):
     status_code = status.HTTP_415_UNSUPPORTED_MEDIA_TYPE
     default_detail = 'Invalid filename'
+
+class ProtectedResourceError(APIException):
+    status_code = status.HTTP_409_CONFLICT
+    default_detail = 'Cannot delete MODEL because it is PROTECTED'
```

Comment:
Change the default detail to "Deletion failed because resource is a member of a protected relationship"